elasticdl/python/master/master.py

```diff
@@ -146,6 +146,17 @@ def __init__(self, args):
         # Initialize instance manager
         self.instance_manager = self._create_instance_manager(args)
 
+        self._should_stop = False
+        self._exit_code = 0
+
+    def request_stop(self, err_msg=None):
```

Comment:
Which component will call this Api?

Reply:
Any component. It provides a mechanism to stop master early. In another PR, I will use this API.